src/System.Composition.Hosting/src/Resources/Strings.resx

```diff
@@ -87,7 +87,7 @@
     <value>Metadata value circularity not possible, use lazy initialization.</value>
   </data>
   <data name="CardinalityMismatch_TooManyExports" xml:space="preserve">
-    <value>"Only one export for the contract '{0}' is allowed, multiple implementations were found."</value>
+    <value>More than one export was found that matches the constraint: {0}</value>
```

Comment:
@weshaggard is it correct to use 'constraint' instead of 'contract' here? (This to match the string in S.C.Composition)

Reply:
I think it depends on what is in `{0}`, I think it is different between the 2 different MEF versions. From a quick look:
https://github.com/dotnet/corefx/blob/fed2d5ff6842b2ce932c70162f3e06a8d0304030/src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Hosting/ExportProvider.cs#L114
and
https://github.com/dotnet/corefx/blob/bffef76f6af208e2042a2f27bc081ee908bb390b/src/System.Composition.Hosting/src/System/Composition/Hosting/Core/ExportDescriptorRegistry.cs#L50

They are different types because of the different MEF versions.

Without digging too much deeper I would say to just rename the key in one of the 2 versions and keep the string separate.

Reply:
done